src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -13,11 +13,19 @@ import com.jetbrains.python.psi.PyFormattedStringElement
 import com.jetbrains.python.psi.PyStringLiteralExpression
 import com.jetbrains.python.psi.types.TypeEvalContext
 
+/**
+ * If [searchRelativelyToCurrentFolder] is true - the file sought relative to the current folder
+ * else relative to the project root
+ * If [makePathRelativelyCurrentFolder] is true - the path built relative to the current folder
```

Comment:
Reply to https://github.com/JetBrains-Research/snakecharm/pull/376#discussion_r667647451

> We have for example "conda:" field, where we search file from root, but path is making from current folder:
"../envs/snpsift.yaml"
Therefore, we need 2 flags to search for a file and build a path

Seems I don't understand the difference. `../envs/snpsift.yaml` means a path relative to the current file folder for file structure:
```
|- envs
|---- snpsift.yaml
|-rules
|---- rule_file.smk
```

Reply:
Since snakemake is looking for this path from the project root, then ../ will force it to exit from the root folder

Reply:
And in this case it looks for global files, but from the current folder

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/377/commits/4fb50285f6868a8c1e12436cb34795b1cb377447